src/Comments.jsx

```diff
@@ -37,6 +39,20 @@ export default class _Comments extends React.Component {
 
   componentDidMount() {
     this._ismounted = true;
+    fetch(`${CUSTOMIZR_URL}/v1/settings?resourceId=http://comment.trdlnk.cimpress.io/`, {
+      method: "GET",
+      headers: {
+        'Authorization': `Bearer ${this.props.accessToken}`,
+      },
+    }).then(res => {
+      return res.json();
+    }).then(json => {
+      if (this._ismounted) {
+        this.setState({alertDismissed: json.data
```

Comment:
In general I would advise to move the calculation of the flag to a separate variable. Makes the code cleaner. But it's not a big deal.

Reply:
this is not calculation but more checking for the exception if one of the fields in the nested json structure would be null. 

I could not check for all the nested objects and catch exception (but that's also weird). It would be nice if we handsome sort of c#ish (or typescryptish) syntax of variable1?.nestedVar1?.nestedVar2.

Either way I wouldn't classify this as logic - the value of the field is copied 1 to 1 if present.

Reply:
It's coming! https://github.com/tc39/proposal-optional-chaining
We can use it today with Babel 7 as well.